spec/system/candidate_interface/entering_details/degrees/new_degree_flow/backlinks_spec.rb

```diff
@@ -0,0 +1,199 @@
+    then_i_am_taken_back_to_the_degree_review_page
+
+    when_i_click_to_change_my_undergraduate_degree_type
+    and_i_click_on_save_and_continue
+    then_i_am_taken_to_the_specific_degree_type_page
+    and_i_click_the_back_link
+    then_i_am_taken_back_to_the_degree_type_page
+    and_i_click_the_back_link
+    then_i_am_taken_back_to_the_degree_review_page
+
+    when_i_click_to_change_my_completion_status
+    and_i_click_on_save_and_continue
+    and_i_click_the_back_link
+    then_i_am_taken_back_to_the_degree_complete_page
+    and_i_click_the_back_link
+    then_i_am_taken_back_to_the_degree_review_page
+
+    when_i_click_to_change_my_university
+    and_i_click_the_back_link
+    then_i_am_taken_back_to_the_degree_review_page
+
+    when_i_click_to_change_my_country
+    and_i_choose_another_country
+    and_i_click_on_save_and_continue
+    and_i_fill_in_a_subject
+    and_i_click_on_save_and_continue
+    and_i_click_the_back_link
+    then_i_am_taken_back_to_the_subject_page
+    and_i_click_the_back_link
+    then_i_am_taken_back_to_the_country_page
+    and_i_click_the_back_link
+    then_i_am_taken_back_to_the_degree_review_page
+
+    given_that_i_have_a_completed_international_degree
+    when_i_view_the_degree_section
+    and_i_click_to_change_my_undergraduate_degree_completion_status
+    when_i_choose_yes
+    and_i_click_on_save_and_continue
+    then_i_am_taken_to_the_award_year_page
+    and_i_fill_out_the_year
+    and_i_click_on_save_and_continue
+    then_i_am_taken_to_the_enic_page
+    and_i_click_the_back_link
+    then_i_am_taken_to_the_award_year_page
+    and_i_click_the_back_link
+    then_i_am_taken_to_the_degree_complete_page
+    and_i_click_the_back_link
+    then_i_am_taken_back_to_the_degree_review_page
+  end
+
+  def given_i_am_signed_in
+    @candidate = create(:candidate)
+    login_as(@candidate)
+  end
+
+  def and_i_have_completed_the_degree_section
+    @application_form = create(:application_form, candidate: @candidate)
+    create(:application_qualification,
+           level: 'degree',
+           qualification_type: 'Bachelor of Arts',
+           start_year: '2006',
+           award_year: '2009',
+           predicted_grade: false,
+           subject: 'Computer Science',
+           institution_name: 'University of Cambridge',
+           institution_country: nil,
+           grade: 'A',
+           application_form: @application_form)
+    @application_form.update!(degrees_completed: true)
+  end
+
+  def when_i_view_the_degree_section
+    visit candidate_interface_application_form_path
+    when_i_click_on_degree
+  end
+
+  def when_i_click_on_degree
+    click_link 'Degree'
+  end
+
+  def and_i_click_to_change_my_undergraduate_degree_type
+    click_change_link('qualification')
+  end
+
+  def and_i_click_the_back_link
+    click_link 'Back'
+  end
+
+  def then_i_am_taken_back_to_the_degree_review_page
+    expect(page).to have_content(t('page_titles.degree'))
```

Comment:
Do you think we should use the actual string rather than the translations in the test like you've done for the rest? It makes the test more readable and also we will catch if we accidently edit the wrong translation in the future.

Reply:
+1 for using actual text in test files (although not everyone agrees 😉)!